www/artists/dloran/ContentFlow/contentflow.js

```diff
@@ -1,2 +1 @@
(s._currentPosition=-.01,s._targetPosition=0,s.resize()):s._initStep()},100),e},rmItem:function(t){if("undefined"==t&&(t=this._activeItem.index),t=this._getIndexByKeyWord(t),!this.items[t])return null;var e=this.items[t];e.pre&&(e.pre.next=e.next),e.next&&(e.next.pre=e.pre),this.items.splice(t,1);for(var i=t;i<this.items.length;i++)this.items[i].setIndex(i);this._setLastIndex(),Math.round(this._getPositionByIndex(t))<Math.round(this._targetPosition)&&(this._targetPosition--,this.conf.circularFlow||(this._targetPosition=this._checkIndex(this._targetPosition))),this._getPositionByIndex(t)<this._currentPosition&&(this._currentPosition--,this.conf.circularFlow||(this._currentPosition=this._checkIndex(this._currentPosition))),this._activeItem=this._getItemByPosition(this._currentPosition);var n=e.element.parentNode.removeChild(e.element),s=this;return window.setTimeout(function(){s._initStep()},10),n},_init:function(){if("string"==typeof this.Container){var t=document.getElementById(this.Container);if(!t)throw"ContentFlow ERROR: No element with id '"+this.Container+"' found!";this.Container=t}$CF(this.Container).addClassName("ContentFlow");var e=$CF(this.Container).getChildrenByClassName("flow")[0];if(!e)throw"ContentFlow ERROR: No element with class'flow' found!";this.Flow=new ContentFlowGUIElement(this,e);var i=this.Container.getChildrenByClassName("scrollbar")[0];if(i){this.Scrollbar=new ContentFlowGUIElement(this,i);var n=this.Scrollbar.getChildrenByClassName("slider")[0];if(n){this.Slider=new ContentFlowGUIElement(this,n);var s=this.Slider.getChildrenByClassName("position")[0];s&&(this.Position=new ContentFlowGUIElement(this,s))}}this.setConfig(this._defaultConf),this._initAddOns(),this.setConfig(this._userConf),this._initSizes();var o=this.Flow.getChildrenByClassName("item");this.items=new Array;for(var h=0;h<o.length;h++){var r=this.items[h]=new ContentFlowItem(this,o[h],h);h>0&&(r.pre=this.items[h-1],r.pre.next=r)}if(this._setLastIndex(),this.conf.circularFlow&&this.items.length>0){var a=this.items[0];a.pre=this.items[this.items.length-1],a.pre.next=a}this._initGUI(),"content"!=this._activeElement&&(this._activeElement="element"),this.conf.origVisibleItems=this.conf.visibleItems,this.conf.visibleItems<0&&(this.conf.visibleItems=Math.round(Math.sqrt(this.items.length))),this.conf.visibleItems=Math.min(this.conf.visibleItems,this.items.length-1),this._targetPosition=this._getIndexByKeyWord(this.conf.startItem,0);var l=this._getIndexByKeyWord(this.conf.scrollInFrom,this._targetPosition);switch(this.conf.scrollInFrom){case"next":case"right":l-=.5;break;case"pre":case"previous":case"left":l+=.5}this._currentPosition=l;var c=new Date,d=this,m=window.setInterval(function(){if(0==d._imagesToLoad||new Date-c>d._loadingTimeout){clearInterval(m),d._activeItem=d.getItem(d._currentPosition),d._activeItem&&(d._activeItem.makeActive(),d._setCaptionLabel(d._activeItem.index)),d.Flow.style.visibility="visible",d.loadIndicator&&(d.loadIndicator.style.display="n
```

Comment:
The minification process seems to have introduced new changes which may make it more confusing as to what exactly was and was not changed to fix the issue. For instance, instead of "window.opera?true:false", we have "!!window.opera". This is just the first of many small changes like this.